packages/Liquid-Core.package/LiquidPoll.class/instance/startWithId.ifTaken..st

```diff
@@ -7,6 +7,6 @@ startWithId: anId ifTaken: do
 		ifPresent: do
 		ifAbsentPut: self.
 		
-	^ PollRepo at: anId ifAbsent: [].
+	^ PollRepo default at: anId ifAbsent: [].
```

Comment:
We also fixed this today in CI-Lint, because the linter complains about this. Maybe for the first time ever it has been useful^^